CHANGES.md

```diff
@@ -120,6 +120,13 @@ breaking changes, and mappings for the large list of deprecated functions.
 
 ### Changes between 3.0.2 and 3.0.3
 
+ * Case insensitive string comparison is reimplemented via new locale-agnostic
+   comparison functions OPENSSL_str[n]casecmp always using POSIX locale for
+   comparison. Previous implementation had severe problems when Turkish locale
```

Comment:
_The previous implementation had problems when the Turkish locale was used._

Reply:
Done, thanks